src/destreamerEvents.ts

```diff
@@ -0,0 +1,26 @@
+import { Errors } from './Types';
+
+import colors from 'colors';
+
+/**
+ * This file contains global destreamer process events
+ *
+ * @note SIGINT event is overridden in downloadVideo function
+ *
+ * @note function is required for non-packaged destreamer, so we can't do better
+ */
+export function setProcessEvents() {
+    process.on('unhandledRejection', (reason) => {
+        console.error(colors.red('Unhandled error!\nTimeout or fatal error, please check your downloads and try again if necessary.\n'));
```

Comment:
Let's improve this to say _...check your downloads **directory** and try again..._